registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java

```diff
@@ -227,8 +228,11 @@ private HttpEntity<Object> setRequestHeader(Object requestType, MediaType mediaT
 				Iterator<String> iterator = httpHeader.keySet().iterator();
 				while (iterator.hasNext()) {
 					String key = iterator.next();
-					if (!(headers.containsKey("Content-Type") && key.equalsIgnoreCase("Content-Type")) && httpHeader.get(key).size()>0)
-						headers.add(key,httpHeader.get(key).get(0));
+					List<String> collection = httpHeader.get(key);
```

Comment:
Sonar was showing bug for previos development (httpHeader.get(key) can return null ) , that's why modifiend in such a way that it will be resolved and logic will not change ..

Reply:
Many of the changes have already been merged in previous pr, only changes in LandingZoneUtility.java and StageClassesUtil.java are yet to be merged